pkg/k8s/apis/cilium.io/v2alpha1/register.go

```diff
@@ -23,7 +23,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.27.0"
+	CustomResourceDefinitionSchemaVersion = "1.26.5"
```

Comment:
I didn't really understand what this was for. I'm happy to write some documentation once I grok it.

Reply:
Here's my understanding: We bump the minor version here once per release series, bump patch ver during a patch version if any CRD changes in any minor way. As far as I understand this is how we upgrade resources, if the number is bigger then when we register the newer version. Release scripting expects the v2 to be same as v2alpha1. A previous commit bumped minor when it didn't need to, so this is walking that back. 

The other solution to what this is solving here would be to change the v2 to match this one. Then we just bumped the minor more times than we needed from one release to the next. I don't think it matters to do that way though because we don't anticipate people upgrading between rcs.